website/docs/plugin/framework/migrating/data-sources/index.mdx

```diff
@@ -0,0 +1,177 @@
+---
+page_title: 'Data Sources: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate a data source from SDKv2 to the plugin Framework.
+---
+
+# Data Sources
+
+Data sources let Terraform reference external data. Unlike resources, Terraform does not create, update, or delete
+data sources, and makes no attempt to modify the underlying API. Data Sources are a read-only resource type, so they
+only implement a subset of the operations that resources do. Refer to [Data Sources](/plugin/framework/data-sources)
+in the Framework documentation for details.
+
+This page explains how to migrate a data source from SDKv2 to the plugin Framework.
+
+## SDKv2
+
+In SDKv2, data sources are defined by the `DataSourcesMap` field on the `schema.Provider` struct, which maps data source
+names (strings) to their schema. The `schema.Resource` struct is used for both resources and data sources.
+
+The following example shows a typical implementation.
+
+```go
+func New() *schema.Provider {
+    return &schema.Provider{
+        DataSourcesMap:   map[string]*schema.Resource{}
```

Comment:
```suggestion
        DataSourcesMap:   map[string]*schema.Resource{
```

Because the next line is supposed to elide the contents of the map definition, right?